index.bs

```diff
@@ -182,16 +182,16 @@ This property should be marked as essential.
 
 <h4 id ="image-spatial-extents-property">Image Spatial Extents Property</h4>
 
-The semantics of the 'ispe' property as defined in [[!HEIF]] apply. More specifically, for AV1 images, it corresponds to the [=FrameWidth=] and [=FrameHeight=] values as defined in [[!AV1]] for a frame that depends on the presence of the 'lsel' and [=OperatingPointSelectorProperty=] properties as follows:
+The semantics of the 'ispe' property as defined in [[!HEIF]] apply. More specifically, for AV1 images, it corresponds to the [=FrameWidth=] and [=FrameHeight=] values as defined in [[!AV1]] for a frame that depends on the presence and content of the 'lsel' and [=OperatingPointSelectorProperty=] properties as follows:
```

Comment:
This sentence feels kind of hard to read, but I'm not completely sure how to improve it. Maybe something like this?

```suggestion
The semantics of the 'ispe' property as defined in [[!HEIF]] apply. More specifically, for AV1 images, it corresponds to the [=FrameWidth=] and [=FrameHeight=] values as defined in [[!AV1]] for the frame selected by the presence and content of the 'lsel' and [=OperatingPointSelectorProperty=] properties as follows:
```